department-of-veterans-affairs/caseflow
# Cleaning up permissions for organizations

### Description
1) Removing the feature toggle entry in the organization table since we don't use it.
2) Allowing System Admins to access organization pages.

### Testing Plan
1. Log in as a system admin ensure that you can access the organization users page
1. Log in as an organization user and ensure you can access the organization page still
1. Log in as a non-organization user and ensure you cannot access the organization page
